### PR TITLE
Script to add raw accel to startup

### PR DESCRIPTION
**Pull request by zdhenard42**
This script will delete all older version of rawaccel shortcuts in startup dir then create a new shortcut in users startup dir

### CODE REVIEW

**Comment by a1xd** [comment]
wildcard in `del` seems a bit too general, otherwise looks good to me!

File: rawaccelstartup.bat [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@echo off
+
+del "%USERPROFILE%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\raw*"
```
Comment (a1xd) [inline rawaccelstartup.bat:3]:
```suggestion
del "%USERPROFILE%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\rawaccel*" 

```

Comment (a1xd) [inline rawaccelstartup.bat:3]:
```suggestion
del "%USERPROFILE%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\rawaccel*" 
```

**Comment by zdhenard42** [comment]
I agree, accepted changes.

**Comment by chrisheib** [comment]
I've had the best success with autostarting rawaccel with adding the `writer.exe` with argument `settings.json` to the windows scheduler with trigger AtLogon. 

Should be achievable with a small powershell script: https://docs.microsoft.com/en-us/powershell/module/scheduledtasks/new-scheduledtask?view=windowsserver2022-ps

Maybe this would be a better solution / alternative for users, where the normal autostart fails?

**Comment by zdhenard42** [comment]
The only issue with that is the first time you run RawAccel through scheduler, a new settings file will be generated wherever scheduled tasks are stored. Once user reenters settings, it will continue to work as normal. If an update is released and user does not place updated release in same dir path as previous version they would have to create a new task and delete the old task.

**Pull request closed without merging**